src/docs/devices/Sonoff-TH10/index.md

```diff
@@ -1,76 +1,93 @@
 ---
-title: Sonoff TH10
+title: Sonoff TH10/TH16
 date-published: 2019-10-11
-type: plug
-standard: au
+type: relay
+standard: global
 board: esp8266
 ---
 
+## Information
+
+The Sonoff TH10 and TH16 are 10A/16A smart WiFi switches with optional temperature and humidity monitoring.
+
 ## GPIO Pinout
 
 | Pin    | Function                           |
 | ------ | ---------------------------------- |
 | GPIO0  | Push Button (HIGH = off, LOW = on) |
 | GPIO12 | Relay and Red LED                  |
 | GPIO13 | Blue Status LED                    |
+| GPIO14 | 1-Wire Bus for 2.5 mm jack         |
 
 ## Basic Configuration
 
+The following configuration also contains commented lines for adding a 1-Wire sensor via the 2.5 mm jack.
+
 ```yaml
+# These substitutions allow for easy renaming
+substitutions:
+  name: "sonoff-th1X"
+  friendly_name: "Sonoff TH1X"
+
 # Basic Config
 esphome:
-  name: sonoffth10_1
+  name: "${name}"
+  friendly_name: "${friendly_name}"
   platform: ESP8266
   board: esp01_1m
   board_flash_mode: dout
-  on_boot:
-    priority: -10
-    then:
-      - light.turn_on: blue_led
-wifi:
-  ssid: "ssid"
-  password: "wifi_password"
-  ap:
-    ssid: "ap_ssid"
-    password: "ap_password"
-  domain: ".domain.com"
 
 # Enable logging
 logger:
 
 # Enable Home Assistant API
 api:
-  encryption:
-    key: !secret api_encryption_key
 
+# Enable the web server
+web_server:
+
+# Enable over-the-air updates
 ota:
-  password: "ota_password"
+  platform: esphome
 
-web_server:
+# Wifi Setup
+wifi:
+  ssid: !secret wifi_ssid
+  password: !secret wifi_password
+
+# One-Wire for 2.5mm jack sensors
+# one_wire:
+# - platform: gpio
+# pin: GPIO14
+
+sensor:
+  - platform: wifi_signal
+    name: "${friendly_name} WiFi Signal"
```

Comment:
Entities should not be prefixed with friendly name substitution as it is added automatically by using `esphome` -> `friendly_name`

```suggestion
    name: WiFi Signal
```